### DISCUSSION TITLE
I made this works on cloudflare pages, with python 3.7

### DISCUSSION THREAD

**Discussion by lgh06** [obsidian-html/obsidian-html#229] [Show and tell]
Since cloudflare pages only support python 3.7, see [here](https://developers.cloudflare.com/pages/platform/build-configuration#language-support-and-tools), so I modify some lines and finally it works on cf pages.  

My fork repo: https://github.com/lgh06/obsidian-html
my fork compared with obsidian-html:  https://github.com/obsidian-html/obsidian-html/compare/5a040df...lgh06:20220426  
Demo: https://kb.hapleo.com/  &  https://daniel-kb.pages.dev/  
My obsidian repo: https://github.com/lgh06/Daniel_kb  .

I am a newbee of Python and I modify python codes by googling and stackoverflow, it is just worked… not fully tested.

**Comment by lgh06** [comment]
build command:
```
python3 -m pip install --upgrade pip && python3 -m pip install https://github.com/lgh06/obsidian-html/archive/refs/heads/master.zip && obsidianhtml -i config_cf.yaml
```
![image](https://user-images.githubusercontent.com/1688639/165228882-be5ee621-500b-4bec-8deb-870f69a11b6b.png)

env:  
```
PYTHON_VERSION  3.7
```
![image](https://user-images.githubusercontent.com/1688639/165229498-a5cb5ae2-50bf-4f66-bd5b-a5c2f68330ad.png)

**Comment by dwrolvink** [comment]
Really cool, nice work.
I hope you can still pull updates from master because version 2.0.2 is still a bit buggy in the graph view department, working hard on it at the moment.

Very tempting to pull back your changes, but I see that they are already working on a new build image [link](https://community.cloudflare.com/t/feature-request-publish-cloudflare-pages-build-system-version-and-add-a-newer-system-for-choosing/368392/2) and seeing how it would be regressing to a considerably older version its better to wait on that for me.

Cool that you managed to fix this for your usecase though, and nice for other people who have the same problem :+1:
